packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -127,6 +128,21 @@ Future<void> buildWindows(
     'Built ${globals.fs.path.relative(buildOutput.path)}',
     color: TerminalColor.green,
   );
+  final File shorebirdYamlFile = buildDirectory
+    .childDirectory('runner')
+    .childDirectory(sentenceCase(buildModeName))
+    .childDirectory('data')
+    .childDirectory('flutter_assets')
+    .childFile('shorebird.yaml');
+
+  if (shorebirdYamlFile.existsSync()) {
+    try {
+      updateShorebirdYaml(buildInfo, shorebirdYamlFile.path, environment: globals.platform.environment);
```

Comment:
I'm confused how anything was working if the shorebird.yaml wasn't being "recompiled"?  My understanding was that the checked in shorebird.yaml was no longer the same as the distributed one?  I guess it was close enough, just presumably flavors (assuming that's even a thing on windows?) wouldn't have been working?

Reply:
We were just copying the shorebird.yaml as an asset, and the updater is able to parse that. None of the desktop platforms support flavors, afaict (ref: https://docs.flutter.dev/deployment/flavors)